spotify_dl/spotify_dl.py

```diff
@@ -93,10 +97,8 @@ def spotify_dl():
             playlist = get_playlist_name_from_id(args.playlist, current_user_id, sp)
 
         log.info("Saving songs to: {}".format(playlist))
-        download_directory = args.output + '/' + playlist
-        # Check whether directory has a trailing slash or not
-        if len(download_directory) >= 0 and download_directory[-1] != '/':
-            download_directory += '/'
```

Comment:
@iambibhas well now I know why that piece of code is required :)  
![image](https://user-images.githubusercontent.com/25424/28466673-ca9aea18-6e4a-11e7-89a5-47ddf1312715.png)